docs/algorithms/sidb_simulation.rst

```diff
@@ -69,14 +71,21 @@ Temperature Behavior
 
 .. doxygenfunction:: fiction::critical_temperature
```

Comment:
This function does not exist anymore, does it?

You can check your documentation build here: https://fiction--120.org.readthedocs.build/en/120/algorithms/sidb_simulation.html#energy-calculation